tests/x509/x509-req-test.sh

```diff
@@ -146,7 +142,7 @@ rm -f tmp.cert
 run_success "req -new -days 3650 -sha512 -key ./certs/server-key.pem -config ./test.conf -out tmp.cert -x509"
 rm -f tmp.cert
 
-run_success "req -new -newkey rsa:2048 -keyout new-key.pem -config ./test.conf -x509 -out tmp.cert" "test"
```

Comment:
lets keep tests for the password being echo'd in, a new -passout test can be added instead of replacing the existing tests

Reply:
Back to testing with echoes. Done with adding `-passout stdin.`